bibliosearch_backend/api/views.py

```diff
@@ -1,3 +1,101 @@
+import json
 from django.shortcuts import render
 
+#import auth
+from django.contrib.auth import authenticate
+
+from django.contrib.auth import logout as auth_logout
+
+from django.contrib.auth import login as auth_login
+
+from django.http import JsonResponse
+
+#import messages
+from django.contrib import messages
+
+#import redirect
+from django.shortcuts import redirect
+
+#import User
+from django.contrib.auth.models import User
+
+#import BiblioSearchUser
+from api.models import BiblioSearchUser
+
+from django.views.decorators.csrf import csrf_exempt
+
+
+#import decorators
+from django.contrib.auth.decorators import login_required  
+
+
 # Create your views here.
+
+# create login view
+@csrf_exempt
```

Comment:
let's try to make this work without csrf_exempt

not a good practice.

also if we can't accomplish that, update requirements.txt with

```sh
pip freeze > requirements.txt
```

Reply:
I removed @csrf_exempt. Frontend needs to get cookies  from the requests and send requests with appriopriate cookies.